### ISSUE TITLE
Can't "Manage Languages" after reinstall (Linux Mint)

### ISSUE THREAD

**Issue by ReaderGuy42** [manisandro/gImageReader#589]
I just did a reinstall of my OS (Linux Mint 20.3) and also reinstalled gImageReader. Even after I install the tesseract-ocr and tesseract-eng package from my Software Manager, I get the following error:

![image](https://user-images.githubusercontent.com/71895400/163599306-001af862-8e52-41fd-9097-ccdd368538c5.png)

As far as I can tell the OCR'ing and output to PDF with overlay does work (not sure how well, my one test PDF seemed a bit off). 

Any ideas on what I forgot? I assume I just missed something on the install.
Thanks!

**Comment by manisandro** [comment]
The error message really says it all - try installing Gnome Software or any other software management application shipping a PackageKit backend.

**Comment by ReaderGuy42** [comment]
OK, sorry, didn't realize PackageKit was actually the name of the thing. I've installed _Gnome-packagekit_ and when I search for tesseract it shows lots of stuff - but I've already installed the language files for English. Now what?
Thanks!!

**Comment by manisandro** [comment]
You mean that you get the error dialog even if you installed gnome-software?

**Comment by ReaderGuy42** [comment]
That is correct. I have also rebooted the PC in the mean time but get the same error.

**Comment by FiableDotBiz** [comment]
I got a similar issue on Fedora 39 workstation. Gnome "Software" was installed, and launching "sudo dnf install PackageKit" gave me as answer that the package " PackageKit" was already installed, which left me perplexed. "sudo dnf install gnome-packagekit" eventually solved the problem. If gImageReader is aware of the  desktop environment, it should be useful to adapt the message error so that non-geek users know precisely what to do.

**Comment by ArtCodesArt** [comment]
I'm having the same issue and my OS is LMDE 6 (Linux Mint Debian Edition)

Here's how to reprodce it:
1. Install gImageReader via Software Manager
2. Open gImageReader, click on chevron at the top beside "English" > select Manage Languages > the pop-up error occurs.
3. I have PackageKit installed and tesseract as well.

**Workaround / Solution** 
1. The popup gives a clue to check the tesseract path
2. gImageReader > Gear icon (top-right) > click on Preferences
3. Language definitions path is listed as `/usr/share/tesseract-ocr/5/tessdata/`<img width="503" height="552" alt="Image" src="https://github.com/user-attachments/assets/bf6ffdb9-db44-458f-a0b5-141623e07d43" />

4. Navigate to that path and you'll notice you probably only have the files `eng.traineddata` and `osd.traineddata`
    - The reason I have more files there is because I installed Polish and French, which is what I was interested in  

<img width="466" height="250" alt="Image" src="https://github.com/user-attachments/assets/85d6a60b-ed8b-45e7-a832-33d43923f6f0" />

5. In your terminal you can also run `tesseract --list-langs` to see what languages are installed.
6. Visit the [tesseract trained data website](https://github.com/tesseract-ocr/tessdata) to download the language models.
7. Move those models to the tesseract path/folder `/usr/share/tesseract-ocr/5/tessdata/`
    - I couldn't copy-paste using GUI so I had to use the command line to use sudo eg. `sudo move pol.traineddata /usr/share/tesseract-ocr/5/tessdata/pol.traineddata`


Error text for reference:
A session connection to the PackageKit backend is required for managing
system-wide tesseract language packs, but it was not found. This service is,
usually provided by a software-management application such as Gnome
Software. Please install software which provides the necessary Packagekit
interface, use other system package management software to manage the
tesseract language packs directly, or switch to using the user tessdata path
in the configuration dialog.
